backend/tests/core/test_security.py

```
@@ -150,17 +158,16 @@ def test_verify_token_with_expired_token(self):
         # Create a token with very short expiration and wait for it to expire
         # Since create_access_token doesn't support negative values properly,
         # we'll create a token manually with past expiration
-        from jose import jwt
-        from datetime import datetime, timedelta
         import time
-        
+        from datetime import datetime, timedelta
```

Ruff (0.14.11)

Redefinition of unused `datetime` from line 5: `datetime` redefined here

Remove definition: `datetime`

(F811)